PWGCF/TwoParticleCorrelations/Tasks/longrangeCorrelation.cxx

```diff
@@ -423,112 +337,29 @@ struct LongrangeCorrelation {
     return true;
   }
 
-  template <typename TTracks>
-  void fillYieldFt0aGlobal(TTracks tracks, bool mixing)
+  template <int mode, typename TTracks>
+  void fillYield(TTracks tracks, bool mixing)
   {
-    if (mixing) {
-      histos.fill(HIST("Ft0aGlobal/ME/hMult"), tracks.size());
-      for (auto const& triggerTrack : tracks) {
-        histos.fill(HIST("Ft0aGlobal/ME/Trig_etavsphi"), triggerTrack.phi(), triggerTrack.eta());
-        histos.fill(HIST("Ft0aGlobal/ME/Trig_eta"), triggerTrack.eta());
-        histos.fill(HIST("Ft0aGlobal/ME/Trig_phi"), triggerTrack.phi());
-        histos.fill(HIST("Ft0aGlobal/ME/Trig_pt"), triggerTrack.pt());
-      }
-    } else {
-      histos.fill(HIST("Ft0aGlobal/SE/hMult"), tracks.size());
-      for (auto const& triggerTrack : tracks) {
-        histos.fill(HIST("Ft0aGlobal/SE/Trig_etavsphi"), triggerTrack.phi(), triggerTrack.eta());
-        histos.fill(HIST("Ft0aGlobal/SE/Trig_eta"), triggerTrack.eta());
-        histos.fill(HIST("Ft0aGlobal/SE/Trig_phi"), triggerTrack.phi());
-        histos.fill(HIST("Ft0aGlobal/SE/Trig_pt"), triggerTrack.pt());
-      }
-    }
-  }
+    static constexpr std::string_view SubDirSE[] = {"Ft0aGlobal/SE/", "Ft0cGlobal/SE/", "Fv0Global/SE/",
+                                                    "MftGlobal/SE/", "Fv0Mft/SE/"};
+    static constexpr std::string_view SubDirME[] = {"Ft0aGlobal/ME/", "Ft0cGlobal/ME/", "Fv0Global/ME/",
+                                                    "MftGlobal/ME/", "Fv0Mft/ME/"};
```

Comment:
I guess that if two mode classes are considered, `{ME, SE}`, and `{FT0A,FT0C,FV0,MFT,FV0MFT}`, a further unification/simplification might be achieved